analysis/src/CompletionBackEnd.ml

```diff
@@ -992,16 +1002,35 @@ let rec extractRecordType ~env ~package (t : Types.type_expr) =
     | _ -> None)
   | _ -> None
 
-let rec extractVariantType ~env ~package (t : Types.type_expr) =
+type extractedType =
+  | Declared of QueryEnv.t * Type.t Declared.t
+  (* Polyvariants seem to be "inlined" in the type system, so we extract what we need from them here. *)
```

Comment:
It's because they're structural types, so you don't have go chase type declarations the way one would do with nominal types.